pkg/util/set.go

```diff
@@ -21,10 +21,10 @@ import (
 	"sort"
 )
 
-type empty struct{}
+type Empty struct{}
```

Comment:
Godoc why this is public.

A test verifying that it's public is nice but optional.